### DISCUSSION TITLE
onedrive上传一直显示后端处理中，但是网盘里面有文件了

### DISCUSSION THREAD

**Discussion by zhujitop** [AlistGo/alist#969] [Q&A]
网盘程序这边一直显示：后端处理中...
http状态是：524

但是网盘里面，已经有这个文件了，就是文件上传过去了。
只是这边还是一直后端处理中。

这个时候直接刷新网盘列表程序，文件也显示出来了。

这种情况算是正常的吗？

**Comment by xhofe** [comment]
正常 可以自己设置nginx超时时间。
